src/Config/SettingsTab.ts

```diff
@@ -115,5 +115,16 @@ export class SettingsTab extends PluginSettingTab {
                         await this.plugin.saveSettings();
                     });
             });
+
+        new Setting(containerEl)
+            .setName('Provide access keys in dialogs')
+            .setDesc('Whether Obsidian Tasks should provide keyboard shortcuts for form controls in dialog boxes.')
```

Comment:
1. What are they? Will most people know what access keys are? I wouldn't have known were it not for earlier discussion on PRs here...
2. From earlier discussion, my impression is that people might want to turn this off if any of the shortcuts interfere with native keyboard shortcuts, or if they interfere with screen readers. Is it worth noting that?
3. Somehow, changing this option does not require a restart of Obsidian. It takes immediate effect, which is really nice. I wonder what it is about this setting that differs from the earlier ones? It may be simply because a new Edit modal is created on each invocation...

Reply:
I have made the description a bit longer (1. and 2. ) - feel free to improve my description if you have better ideas.

Yes, the setting is re-evaluated every time a new modal is created (3.).

Reply:
I really like the new wording!